modules/core/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "main": "./main.ts",
   "dependencies": {
+    "@dimforge/rapier2d-compat": "^0.12.0",
```

Comment:
this `rapier2d-compat` dependency was easier to get working. under the hood, a WASM file is loaded